api/games/create.js

```diff
@@ -53,7 +55,7 @@ export const createGameFromLobby = gameLobby => {
 
   // Ask (experimenter designer) init function to configure this game
   // given the factors and players given.
-  const params = { data: {}, rounds: [], players };
+  const params = { data: {}, rounds: [], players, gameLobby };
```

Comment:
Here we will want to copy the data in lobby to the game data:

```suggestion
  const params = { data: { ...gameLobby.data }, rounds: [], players };
```